hal/src/nRF52840/mbedtls/sha512_disable.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2023 Particle Industries, Inc.  All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "crys_hash.h"
+
+uint32_t __wrap_InitSwHash512(void* pCtx) {
```

Comment:
Would we be able to do this with SHA1 as well or do we still use it somewhere?

Reply:
We do not need anything other than SHA256 (for cloud and BLE).